wweval/R/plots.R

```diff
@@ -1051,7 +1051,16 @@ get_plot_hub_performance <- function(all_scores,
       "location"
     )) |>
     dplyr::mutate(relative_wis = interval_score / baseline_score) |>
-    dplyr::filter(model != {{ baseline_model }})
+    dplyr::filter(model != {{ baseline_model }}) |>
+    dplyr::mutate(
+      fig_order =
+        dplyr::case_when(
+          period == "Feb 2024-Mar 2024" ~ 1,
+          period == "Oct 2023-Mar 2024" ~ 0
+        ),
+      horizon = forcats::fct_reorder(horizon, fig_order)
+    )
```

Comment:
I assume you meant to order `period` here, not `horizon`, yes? Another case I'd handle with base `factor`:
```suggestion
    dplyr::mutate(
      period = factor(period, ordered = TRUE, levels = c(
          "Feb 2024-Mar 2024", "Oct 2023-Mar 2024"))
    )
```

Reply:
Whoops good catch yes!